portfolio/src/main/webapp/index.html

```diff
@@ -30,6 +30,28 @@ <h1>Heidi Johnson's Portfolio</h1>
             <input type="submit" />
           </form>
           <ul id="comments-container"></ul>
+          
+          <label for="number-comments">Number of comments:</label>
+            <select onclick="getNumCommentsSelected()" name="number-comments" id="number-comments">
+              <option>0</option>
+              <option>1</option>
+              <option>2</option>
+              <option>3</option>
+              <option>4</option>
+              <option>5</option>
+            </select> 
+          <script>
+            document.getElementById('number-comments').onchange = function() {
+              localStorage.setItem('selectedItem', document.getElementById('number-comments').value);
+            };
+
+            if (localStorage.getItem('selectedItem')) {
+              document.getElementById('number-comments').options[localStorage.getItem('selectedItem')].selected = true;
+            }
```

Comment:
Is there a reason for this script to be embedded directly in your HTML page as opposed to living in a separate .js file?